tests/vmi_multus_test.go

```diff
@@ -464,10 +464,12 @@ var _ = Describe("[Serial]Multus", func() {
 					&expect.BSnd{S: fmt.Sprintf("ip addr show eth0 | grep %s | wc -l", interfacesByName["default"].MAC)},
```

Comment:
I think it can't be `ip addr show eth0`, since we don't know which interface has the mac, right?